app/views/users/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1>Here is the description and posts for a given user</h1>
+<%= render "shared/user_box", user: @user %>
+<div class = "bio-box-container">
+  <div class = "bio-box">
+    <p>Bio</p>
+    <br>
+    <p><%=@user.bio%></p>
+  </div>
+</div>
+<% @posts.limit(3).each do |post| %>
```

Comment:
- [ ] Hi @mrcbq, so far you have displayed the last three blog posts of your app(Not For A User) :+1: however, in this milestone, we were told to utilized the methods created when processing data in our models(previous milestone). You name this method as `#recent_posts`. This method should render the **user's 3 most recent post**. We are supposed to reused our methods now. Kindly use this method now :+1:

Reply:
Thank you for your suggestion, It's Done!

Reply:
This change has been done well